src/VisualStudio/Core/Def/Implementation/LanguageClient/VisualStudioInProcLanguageClient.cs

```diff
@@ -41,12 +41,14 @@ internal class PullDiagnosticsInProcLanguageClient : AbstractInProcLanguageClien
         }
 
         public override string Name
-            => ServicesVSResources.CSharp_Visual_Basic_Diagnostics_Language_Client;
+            => ServicesVSResources.CSharp_Visual_Basic_Language_Server_Client;
 
         protected internal override VSServerCapabilities GetCapabilities()
             => new VSServerCapabilities
             {
                 SupportsDiagnosticRequests = _globalOptionService.GetOption(InternalDiagnosticsOptions.NormalDiagnosticMode) == DiagnosticMode.Pull,
+                DisableGoToWorkspaceSymbols = true,
```

Comment:
this ensures that local cntrl+, search uses the old editor apis, so only cntrl+Q is powered via LSP

Reply:
we should doc this in the code.